h/search/core.py

```diff
@@ -58,6 +58,11 @@ def run(self, params):
 
         return SearchResult(total, annotation_ids, reply_ids, aggregations)
 
+    def clear(self):
```

Comment:
I'm open to alternative name suggestions-I would just say clear_filters but technically it clears filters, matchers, and aggregators but to say all that would result in a very long method name. Maybe clear_query?

Reply:
I think `clear()` is a good name - `search.clear()` is clear enough

Reply:
Also the docstring clarifies it perfectly